composer.json

```diff
@@ -14,16 +14,17 @@
     "require": {
         "php": "^5.5.9|^7.0",
         "symfony/framework-bundle": "^2.8|^3.0",
-        "symfony-cmf/slugifier-api": "^1.0"
+        "symfony-cmf/slugifier-api": "^1.0",
+        "symfony/symfony": "3.1.*"
     },
     "require-dev": {
         "symfony/security-bundle": "^2.8|^3.0",
         "mockery/mockery": "^0.9.4",
-        "symfony-cmf/routing-bundle": "^1.2|^2.0",
+        "symfony-cmf/routing-bundle": "^1.4.0|^2.0",
```

Comment:
the problem seems to be the symfony-cmf/routing that we get as 1.4.0-RC1. this comes up now because of the minimum-stability change. can we use `@dev` on the requirements where we want dev versions, instead of generally accepting non-stable versions?